bindings/python/pydeck/tests/fixtures/minimal.json

```diff
@@ -0,0 +1,46 @@
+{
+  "initialViewState": {
+    "bearing": -27.396674584323023,
+    "latitude": 52.232395363869415,
+    "longitude": -1.4157267858730052,
+    "maxZoom": 15,
+    "minZoom": 5,
+    "pitch": 40.5,
+    "zoom": 6.6
+  },
+  "layers": [
+    {
+      "coverage": 1,
+      "data": "https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/3d-heatmap/heatmap-data.csv",
+      "elevationRange": [
+        0,
+        3000
+      ],
+      "elevationScale": 50,
+      "extruded": true,
+      "getPosition": "[lng, lat]",
+      "id": "heatmap",
+      "lightSettings": {
+        "ambientRatio": 0.4,
+        "diffuseRatio": 0.6,
+        "lightsPosition": [
+          -0.144528,
+          49.739968,
+          8000,
+          -3.807751,
+          54.104682,
+          8000
+        ],
+        "numberOfLights": 2
+      },
+      "@@type": "HexagonLayer"
```

Comment:
I'd list this as the first prop. Unless this is automatically reordered by some linter.

Reply:
The keys are currently alphabetized for simplicity

Reply:
Well, `@` comes before `A` in the ASCII order https://www.ascii-code.com/ ;)

Reply:
This is an excellent point